### PR TITLE
Assignment 0

### PR DESCRIPTION
**Pull request by francescoperera**

### CODE REVIEW

File: people/francesco-perera.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Francesco Perera
+------------------
+
+![](photos/francescoperera.jpg)
```
Comment (dblock) [inline people/francesco-perera.md]: Please rename this file to francsco-perera.jpg to be consistent with the rest of the file names.

**Comment by dblock** [comment]
When you amend the changes, make sure to have 1 _squashed_ commit. See http://code.dblock.org/2015/08/31/getting-out-of-your-first-git-mess.html if you need help. Also try to write a better commit description than "I added my files".

**Comment by dblock** [comment]
Almost there. Can you rebase against upstream please and put your name in the list in alphabetical order? Thanks.

**Comment by francescoperera** [comment]
@dblock  I believe I am ok

**Comment by dblock** [comment]
Looks good.

**Pull request merged**